app/assets/stylesheets/modules/auth_link.scss

```diff
@@ -4,7 +4,7 @@
 .#{$namespace}-auth-link {
   @include stanford-only-red;
   background-color: $sul-auth-link-background-color;
-  bottom: 15px;
+  bottom: 24px;
```

Comment:
I could be convinced to change this to a variable (since I think it may exist elsewhere in the css).